thundra/plugins/trace/trace_plugin.py

```diff
@@ -51,6 +52,7 @@ def before_invocation(self, plugin_context):
                                                    trace_id=trace_id,
                                                    transaction_id=transaction_id)
         self.root_span = self.scope.span
+        trace_support.root_span_started = True
```

Comment:
Shouldn't this be reset to `False` after every invocation?

Reply:
Yes it should, will fix it

Reply:
Maybe better approach might be using `get_active_span` over `ThundraTracer`. So if there is any active span, we can proceed instead introducing another variable `root_span_started`.